main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+# Creating a specific eright with ERTP
```

Comment:
Let's call it "non-fungible" rather than "specific". The cryptocurrency world has really embraced the term "non-fungible" - NFTs (non-fungible tokens, aka "nifties") are what people are calling them.

Reply:
ok, thanks for the clarification. I'm still new to the cryptocurrency world and this is something i hadn't caught on yet